target/scripts/update-check.sh

```diff
@@ -41,20 +38,19 @@ There is a docker-mailserver update available on your host: $(hostname -f)
 Current version: ${VERSION}
 Latest  version: ${LATEST}
 
-Changelog: ${CHANGELOG}
+Changelog: ${CHANGELOG_URL}
 EOM
       echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && \
 
-      _log "Info: Update available [ ${VERSION} --> ${LATEST} ]" && \
-
       # only notify once
-      exit 0
+      _log_with_date 'info' "Update available [ ${VERSION} --> ${LATEST} ]" && exit 0
     else
-      _log "Info: No update available"
+      _log_with_date 'debug' 'No update available'
```

Comment:
This should also be `info`, so that once a day, there is a log line with either update available or not.

```suggestion
      _log_with_date 'info' 'No update available'
```